libs/designer/src/lib/core/state/templates/panelSlice.ts

```diff
@@ -33,6 +37,16 @@ export const panelSlice = createSlice({
       state.isOpen = true;
       state.currentPanelView = TemplatePanelView.CreateWorkflow;
     },
+    openConfigureWorkflowPanelView: (state) => {
```

Comment:
Instead of having separate actions for each panel, can we just send the panel view in action payload so from the caller? or from a method to simplify calling from ux components? the stores here just have many small methods.